Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -162,6 +164,7 @@ private void OnPlayersSpawned(RulePlayerJobsAssignedEvent ev)
             _traitors.Add(traitorRole);
         }
 
+        TotalTraitors = _traitors.Count;
```

Comment:
Why isn't this just a property pointing to count?

Reply:
If you can tell me what that means I'll add it